### ISSUE TITLE
medaka command error

### ISSUE THREAD

**Issue by JYLeeBioinfo** [rvolden/Mandalorion#3]
I temporarily resolved the previous issue(#2 ) by adding unique numbers starting from 0

but I bumped into another issue of medaka as following
```
Traceback (most recent call last):
  File "createConsensi.py", line 318, in <module>
    main()
  File "createConsensi.py", line 303, in main
    determine_consensus(name, fasta, fastq, str(counter))
  File "createConsensi.py", line 273, in determine_consensus
    temp_folder + '/' + counter, temp_folder + '/' + counter)

sError: Command 'medaka -f -i  mp/1_subsampled.fastq -d  mp/1_consensus_1.fasta -o mp/1 > mp/1_medaka_messages.txt 2>&1' returned non-zero exit status 2
```

(I changed os.system to subprocess.run to propagate exceptions )

directly running the medaka command gave this error
```
usage: medaka [-h] [--version]
              {compress_bam,features,train,consensus,smolecule,consensus_from_features,fastrle,stitch,variant,snp,tools}
              ...
medaka: error: argument command: invalid choice: 'mp/1_subsampled.fastq' (choose from 'compress_bam', 'features', 'train', 'consensus', 'smolecule', 'consensus_from_features', 'fastrle', 'stitch', 'variant', 'snp', 'tools')
```

I initially installed medaka using this command
```
conda create -n medaka -c conda-forge -c bioconda medaka
```

**Comment by JYLeeBioinfo** [comment]
The issue is partly resolved by setting medaka to medaka_consensus

But the following error message arose

```
Running medaka consensus
[19:22:19 - Predict] Setting tensorflow inter/intra-op threads to 1/1.
[19:22:19 - Predict] Processing region(s): chr15_5l1057-3r1077~5l1056-3r1076~5l1055-3r1075~_+_44711489_44718159_0.0_58.0:0-1002
[19:22:19 - Predict] Using model: lib/python3.6/site-packages/medaka/data/r941_min_high_g360_model.hdf5.
[19:22:19 - ModelStore] ModelStore exception <class 'RuntimeError'>
Traceback (most recent call last):
  File "bin/medaka", line 11, in <module>
    sys.exit(main())
  File "lib/python3.6/site-packages/medaka/medaka.py", line 684, in main
    args.func(args)
  File "lib/python3.6/site-packages/medaka/prediction.py", line 123, in predict
    model_store.copy_meta(args.output)
  File "lib/python3.6/site-packages/medaka/datastore.py", line 92, in copy_meta
    return ds.copy_meta(other)
  File "lib/python3.6/site-packages/medaka/datastore.py", line 253, in copy_meta
    name=self._meta_group_)
  File "lib/python3.6/site-packages/h5py/_hl/group.py", line 495, in copy
    copypl, base.dlcpl)
  File "h5py/_objects.pyx", line 54, in h5py._objects.with_phil.wrapper
  File "h5py/_objects.pyx", line 55, in h5py._objects.with_phil.wrapper
  File "h5py/h5o.pyx", line 217, in h5py.h5o.copy
RuntimeError: Unable to copy object (destination object already exists)
Failed to run medaka consensus.
```


But directly running the same medaka code from bash command line is working
```
Running medaka consensus
[21:11:32 - Predict] Setting tensorflow inter/intra-op threads to 1/1.
[21:11:32 - Predict] Processing region(s): chrM_M5_+_7584_8092_6.0_0.0:0-1028
[21:11:32 - Predict] Using model: lib/python3.6/site-packages/medaka/data/r941_min_high_g360_model.hdf5.
[21:11:33 - Predict] Processing 1 long region(s) with batching.
[21:11:33 - ModelStore] filepath lib/python3.6/site-packages/medaka/data/r941_min_high_g360_model.hdf5
[21:11:33 - DLoader] Initializing data loader
[21:11:33 - Sampler] Initializing sampler for consensus of region chrM_M5_+_7584_8092_6.0_0.0:0-1028.
[21:11:33 - PWorker] Running inference for 0.0M draft bases.
[21:11:33 - Feature] Processed chrM_M5_+_7584_8092_6.0_0.0:0.0-1027.0 (median depth 2.0)
[21:11:33 - Sampler] Took 0.09s to make features.
[21:11:33 - Sampler] Region chrM_M5_+_7584_8092_6.0_0.0:0.0-1027.0 (1042 positions) is smaller than inference chunk length 10000, quarantining.
[21:11:34 - PWorker] All done, 1 remainder regions.
[21:11:34 - Predict] Processing 1 short region(s).
[21:11:34 - ModelStore] filepath lib/python3.6/site-packages/medaka/data/r941_min_high_g360_model.hdf5
[21:11:34 - DLoader] Initializing data loader
[21:11:34 - Sampler] Initializing sampler for consensus of region chrM_M5_+_7584_8092_6.0_0.0:0-1028.
[21:11:34 - PWorker] Running inference for 0.0M draft bases.
[21:11:34 - Feature] Processed chrM_M5_+_7584_8092_6.0_0.0:0.0-1027.0 (median depth 2.0)
[21:11:34 - Sampler] Took 0.09s to make features.
[21:11:36 - PWorker] All done, 0 remainder regions.
[21:11:36 - Predict] Finished processing all regions.
Polished assembly written to mp/1/consensus.fasta, have a nice day.
```

**Comment by JYLeeBioinfo** [comment]
I think I found the reason

![image](https://user-images.githubusercontent.com/13775022/108586348-27326b80-7391-11eb-949c-9063e3e35c9d.png)

Due to this part of createConsensi.py, the same job is running twice at the same time.

So one of the job throw error when creating hdf5 file since the other job is already using it.

Is it okay to mute 302 line?

**Comment by rvolden** [comment]
Yeah that should be fine

**Comment by JYLeeBioinfo** [comment]
Thank you!

I tried it after I posted that and it seems to be running well in parallel without error.
